pkg/nodeconfig/nodeconfig.go

```diff
@@ -214,10 +219,9 @@ func (nc *NodeConfig) Configure(ctx context.Context) error {
 			return fmt.Errorf("error waiting for proper %s annotation for node %s: %w", metadata.VersionAnnotation,
 				nc.node.GetName(), err)
 		}
-
 		// Now that the node has been fully configured, update the node object in NodeConfig once more
 		if err := nc.setNode(ctx, false); err != nil {
-			return fmt.Errorf("error getting node object: %w", err)
+			return fmt.Errorf("error setting node object: %w", err)
```

Comment:
nit: Good change, really should be in a separate commit, not worth the effort to change it for this PR, but try to do so in the future.